src/__mocks__/@/lib/db.js

```
@@ -0,0 +1,42 @@
+const { PrismaClientKnownRequestError } = require('@prisma/client/runtime/library');
```

ESLint

[error] 'PrismaClientKnownRequestError' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)